src/components/ui/button.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { Slot } from "@radix-ui/react-slot";
+import { cva, type VariantProps } from "class-variance-authority";
+
+import { cn } from "@/lib/utils";
+
+const buttonVariants = cva(
+  "inline-flex items-center justify-center gap-2 whitespace-nowrap rounded-md text-sm font-medium transition-all disabled:pointer-events-none disabled:opacity-50 [&_svg]:pointer-events-none [&_svg:not([class*='size-'])]:size-4 shrink-0 [&_svg]:shrink-0 outline-none focus-visible:border-ring focus-visible:ring-ring/50 focus-visible:ring-[3px] aria-invalid:ring-destructive/20 dark:aria-invalid:ring-destructive/40 aria-invalid:border-destructive",
+  {
+    variants: {
+      variant: {
+        primary:
+          "bg-[var(--color-green-800)] text-white hover:bg-[var(--color-green-900)] rounded-md py-2 px-4 gap-[10px] min-w-[105px] min-h-[40px] font-sans text-sm leading-6",
```

Comment:
We don't need to use the var here

```suggestion
          "bg-green-800 text-white hover:bg-green-900 rounded-md py-2 px-4 gap-[10px] min-w-[105px] min-h-[40px] font-sans text-sm leading-6",
```

Reply:
This appears again on this file, didn't pointed out, but they also should be changed